gtk2_ardour/luawindow.cc

```diff
@@ -133,9 +133,9 @@ LuaWindow::LuaWindow ()
 
 	// layout
 
-	Gtk::ScrolledWindow *scrollin = manage (new Gtk::ScrolledWindow);
-	scrollin->set_policy (Gtk::POLICY_AUTOMATIC, Gtk::POLICY_AUTOMATIC);
-	scrollin->add (entry);
+	Gtk::ScrolledWindow *scrolling = manage (new Gtk::ScrolledWindow);
+	scrolling->set_policy (Gtk::POLICY_AUTOMATIC, Gtk::POLICY_AUTOMATIC);
+	scrolling->add (entry);
 	scrollout.set_policy (Gtk::POLICY_NEVER, Gtk::POLICY_ALWAYS);
 	scrollout.add (outtext);
```

Comment:
Duh, just realized these lines give it better context scroll**in** and scroll**out**  
I guess I'll just close this ticket then?